<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AVMediaPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/FILES/assets/ProgramIcon.png</normaloff>:/FILES/assets/ProgramIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QListView" name="listView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnPTrack">
            <property name="text">
             <string>&lt;&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FILES/assets/StopIcon.png</normaloff>:/FILES/assets/StopIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNTrack">
            <property name="text">
             <string>&gt;&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMute">
            <property name="text">
             <string>Mute</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderVolume">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblVolume">
            <property name="text">
             <string>100%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnPlay">
            <property name="text">
             <string>Play</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/FILES/ICONS/assets/PlayIcon.png</normaloff>:/FILES/ICONS/assets/PlayIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPause">
            <property name="text">
             <string>Pause</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FILES/assets/PauseIcon.png</normaloff>:/FILES/assets/PauseIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnShuffle">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Shuffle</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FILES/assets/ShuffleIcon.png</normaloff>:/FILES/assets/ShuffleIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnFullscreen">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fullsceen</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/FILES/assets/FullscreenIcon.png</normaloff>:/FILES/assets/FullscreenIcon.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="progressSlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTimeStamp">
            <property name="text">
             <string>////</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionURL"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionplay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionHide_List"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
    <addaction name="actionMute"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
   <addaction name="menuAudio"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionplay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionURL"/>
   <addaction name="actionFullscreen"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionplay">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/PlayIcon.png</normaloff>:/FILES/assets/PlayIcon.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/PauseIcon.png</normaloff>:/FILES/assets/PauseIcon.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/StopIcon.png</normaloff>:/FILES/assets/StopIcon.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/OpenIcon.png</normaloff>:/FILES/assets/OpenIcon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionURL">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/URLIcon.png</normaloff>:/FILES/assets/URLIcon.png</iconset>
   </property>
   <property name="text">
    <string>Open URL</string>
   </property>
   <property name="toolTip">
    <string>Open URL stream</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/FILES/assets/FullscreenIcon.png</normaloff>:/FILES/assets/FullscreenIcon.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear playlist</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="toolTip">
    <string>Mute/Unmute audio</string>
   </property>
  </action>
  <action name="actionHide_List">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide List</string>
   </property>
   <property name="toolTip">
    <string>Hide queue</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
